### ISSUE TITLE
ftp-electron status?

### ISSUE THREAD

**Issue by ghost** [AvraamMavridis/ftp-electron#1]
Hi is this application still in development?

**Comment by AvraamMavridis** [comment]
@di48l069 yep, I am working on that, I guess it will take 3-4 weeks till have something solid :)

**Comment by ghost** [comment]
Do you need some cool design ideas? also i'm more than happy to test

**Comment by AvraamMavridis** [comment]
@di48l069 definitely, any design idea would be highly appreciated

**Comment by ghost** [comment]
Epic I will try whip up something cool tonight... Code wise though I only do
c# and wpf

**Comment by AvraamMavridis** [comment]
:)

**Comment by roccomuso** [comment]
If test is needed, I could help ;)

**Comment by AvraamMavridis** [comment]
I wish I found some time to complete this project

**Comment by PixellUp** [comment]
I was searching on Google how to create ftp client on electron and I found this, but looking at the code there isn't any ftp logic code inside. What is your plan to connect to ftp server, when you can't do it with javascript?

**Comment by AvraamMavridis** [comment]
@doreto currenlty I am quite buzy to work on that. I started it as a side project... but never had time to actually commit on it.

**Comment by samuel-allan** [comment]
I suggest building on top of [steelbrain/node-ssh](https://github.com/steelbrain/node-ssh).
It is a very good library with support for promises, and it's actively maintained (unlike simple-ssh for instance).

**Comment by samuel-allan** [comment]
I would be more than happy to help with the node FTP logic, so feel free to request any help if needed.
